protocol/operation.rei

```diff
@@ -69,15 +61,4 @@ module Side_chain: {
       ~kind: kind
     ) =>
     t;
-
-  let verify:
-    (
-      ~hash: BLAKE2B.t,
-      ~signature: Protocol_signature.t,
-      ~nonce: int32,
-      ~block_height: int64,
-      ~source: Address.t,
-      ~kind: kind
-    ) =>
-    result(t, string);
```

Comment:
This function should not be used, and it's not, that's why I removed it from the interface.